Added documentation for optimizer-rules-getter (#947)
arangodb/docs: 3.10/release-notes-api-changes310.md

## HTTP RESTful API

> Added a GET request for route `/_api/query/rules` that returns the available optimizer rules for AQL queries. This returns an array of objects which display the name of each rule available and its respective flags, wheter set to true or false. 

Reviewer: ```suggestion
Added a GET request for route `/_api/query/rules` that returns the available optimizer rules for AQL queries. This returns an array of objects which contain the name of each rule available and its respective flags. 
```
